frontend/src/App.js

```diff
@@ -2,22 +2,29 @@ import './App.css';
 import ProductPage from './pages/productpage/ProductPage';
 import Navbar from './components/navbar/Navbar';
 import Home from './pages/homepage/Home';
+import Signup from './pages/signup/Signup';
 import Bottombar from './components/bottombar/Bottombar';
```

Comment:
suggestion: you can use absolute import here , It will reduce unnecessarily lines of code and improve your code structure , and helps in understanding the path of files .

Reply:
thanks for the suggestion , I will change it into absolute import.